app/shared/src/commonMain/kotlin/ui/subject/episode/EpisodeVideo.kt

```diff
@@ -425,3 +447,10 @@ internal fun EpisodeVideoImpl(
         },
     )
 }
+
+
+enum class SideSheetState {
```

Comment:
```suggestion
private enum class SideSheetState {
```

`SideSheetState` 名字很通用, 如非必要不要 public. public 会污染全局命名空间